src/convertToObject.js

```diff
@@ -3,16 +3,26 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const resObj = sourceString.replace(/\s+/g, ' ')
+    .split(';').filter(function(entry) {
+      return /\S/.test(entry);
```

Comment:
cmon, don't just copy and paste
what is entry in this case, name it properly
use arrow function
or just rewrite in more simple and readeble way

![Screenshot 2022-09-01 at 15 45 11](https://user-images.githubusercontent.com/99331793/187916959-83d829e1-c9f3-4101-8f41-3336c056d33a.png)